projects/batfish/src/test/java/org/batfish/bdp/BdpDataPlanePluginTest.java

```diff
@@ -177,4 +177,59 @@ public void testIbgpRejectOwnAs() throws IOException {
      */
     assertFalse(r2bPrefixes.contains(r1Loopback0Prefix));
   }
+
+  @Test
+  public void testEbgpAcceptSameNeighborID() throws IOException {
+    String testrigName = "ibgp-only-reject-routerid-match";
```

Comment:
The name of this testrig is not meaningfully different from the name of the other testrig. Perhaps name them after the test function names?